addons/website/static/src/snippets/s_popup/options.js

```diff
@@ -79,6 +79,12 @@ options.registry.SnippetPopup = options.Class.extend({
             this.$target.modal('hide');
         });
     },
+    /**
+     * @override
+     */
+     cleanForSave: function () {
```

Comment:
Misindentation here :open_mouth: If you use VS code, this is a known problem with the "full" auto-indent feature

![image](https://user-images.githubusercontent.com/10338094/146571392-15798df9-7886-43ad-acdd-3a6c9a48ff22.png)

You can fix it by switching to "advanced" (I personally did it only for javascript files):
```
    "[javascript]": {
        "editor.autoIndent": "advanced"
    },
```

Reply:
:+1: